V 2.1.2

ccuilla opened the pull request:
This updates all of the examples to the latest Meteor patch.

Also, more importantly, exports the helper functions.

ccuilla committed:
Updated example apps; Fixed autorun disable feature

   .versions
   client/startup.js
   examples/flow-router/.meteor/.finished-upgraders
   examples/flow-router/.meteor/packages
   examples/flow-router/.meteor/release
   examples/flow-router/.meteor/versions
   examples/flow-router/package.json
   examples/flow-router/settings.json
   examples/iron-router/.meteor/.finished-upgraders
   examples/iron-router/.meteor/packages
   examples/iron-router/.meteor/release
   examples/iron-router/.meteor/versions
   examples/iron-router/package.json
   examples/iron-router/settings.json
   examples/react-router/.meteor/packages
   examples/react-router/.meteor/release
   examples/react-router/.meteor/versions
   examples/react-router/package.json
   examples/react-router/settings.json
   package.js

ccuilla committed:
Updated example app packages; Added package.json for dev stuff; tweaked npm start script in examples

   examples/flow-router/.meteor/versions
   examples/flow-router/package.json
   examples/iron-router/.meteor/versions
   examples/iron-router/package.json
   examples/react-router/.meteor/versions
   package.json

ccuilla committed:
Updated Meteor version for all exmaples; Exported helper functions

   .versions
   client/main.js
   examples/flow-router/.meteor/packages
   examples/flow-router/.meteor/release
   examples/flow-router/.meteor/versions
   examples/iron-router/.meteor/packages
   examples/iron-router/.meteor/release
   examples/iron-router/.meteor/versions
   examples/react-router/.meteor/release
   examples/react-router/.meteor/versions
   package.json

ghost commented:
Oh, also fixed the auto run issue (which I thought was an open issue but appears not to be). Maybe this one: https://github.com/okgrow/analytics/issues/142 ?

Helps with this issue too: https://github.com/okgrow/analytics/issues/152

ccuilla committed:
Back version this to 2.1.1

   .versions
   examples/flow-router/.meteor/versions
   examples/iron-router/.meteor/versions
   examples/react-router/.meteor/versions
   package.json

ghost commented:
So this is my bad. Forgot that there was still an outstanding PR for version 2.1.1. I just moved this one back to v2.1.1. The other PR should get merged and then this one.

cfnelson commented:
👍  Looks good. Mainly bumping the example repos up.
